Lib/test/test_shlex.py

```diff
@@ -337,6 +337,18 @@ def testQuote(self):
             self.assertEqual(shlex.quote("test%s'name'" % u),
                              "'test%s'\"'\"'name'\"'\"''" % u)
 
+    def testQuoteAlways(self):
+        strs = ['hello', 'to the', 'world', 'escape me', 'no-escape-needed']
+
+        # guarantee escaping all strings
+        strs_always_escaped = [shlex.quote(s, always=True) for s in strs]
+        self.assertTrue(all(s.startswith("'") for s in strs_always_escaped))
```

Comment:
I would prefer assertEqual() test to test the whole strings, not just the first character. Same below.

Reply:
Addressed in ffdc6658874efc4fc47d1720d442c5ad08c4cd7b. I've coded in the expected results explicitly; it makes the testing a bit more thorough yes.